google/cloud/security/notifier/notifier.py

```diff
@@ -91,8 +93,53 @@ def _get_timestamp(statuses=('SUCCESS', 'PARTIAL_SUCCESS')):
 
     return latest_timestamp
 
+def process(message):
+    """Process messages about what notifications to send.
+
+    Args:
+        message (dict): Message with payload in dict.
+            The payload will be different depending on the sender
+            of the message.
+
+            Example:
+                {'status': 'foobar_done',
+                 'payload': {}}
+    """
+    payload = message.get('payload')
+
+    if message.get('status') == 'inventory_done':
+        inv_email_pipeline = inv_summary.EmailInventorySnapshopSummaryPipeline(
```

Comment:
EmailInventorySnapshopSummaryPipeline -> EmailInventorySnapshotSummaryPipeline

(Snapshot, not Snapshop)